in_text/text_00_parent.yml

```diff
@@ -6,7 +6,7 @@ abstract: >-
   makers who can use cold-water reservoir releases to maintain thermal habitat for sensitive fish species. 
   This data release contains the forcings and outputs of 7-day ahead maximum water temperature forecasting models
   that makes predictions at 70 river reaches in the upper Delaware River Basin. The modeling approach
```

Comment:
I see mentions of the GLM 3.1 model below. Is this the model that was used for this entire data release? If so, maybe we should mention that in this paragraph and also provide a link to more information about that model.